### PR TITLE
Uc 202 Adding error message and tracking for failed captcha load

### PR DESCRIPTION
**Pull request by lizlux**
In regions like China, where google is blocked, the recaptcha plugin (from google) may fail to load. When this happens we want to inform the user and track the instance so we can get a better idea of when / how often it's happening.

### CODE REVIEW

File: extensions/wikia/UserLogin/css/UserSignup.scss [hunk 1]
```diff
@@ -3,6 +3,7 @@
 @import 'skins/shared/mixins/wordmark';
```
Comment (lizlux) [inline extensions/wikia/UserLogin/css/UserSignup.scss]: Mostly coding convention cleanup in this file

File: extensions/wikia/UserLogin/css/UserSignup.scss [hunk 2]
```diff
@@ -211,3 +212,7 @@
 .WikiaMainContent {
 	float: none;
 }
+
```
Comment (lizlux) [inline extensions/wikia/UserLogin/css/UserSignup.scss]: Here's the actual change

**Comment by garthwebb** [comment]
LGTM

**Comment by jsutterfield** [comment]
LGTM

**Pull request merged**